examples/docs-examples/examples/vpn-network/netcat-basic/netcat_basic.mjs

```diff
@@ -0,0 +1,68 @@
+import dotenv from "dotenv";
+
+import { TaskExecutor, LogLevel } from "@golem-sdk/golem-js";
+
+dotenv.config();
+
+(async function main() {
+  const executor = await TaskExecutor.create({
+    // What do you want to run
+    capabilities: ["vpn"],
+    package: "f4a261ea7b760a1da10f21f0ad8d704c25c8d2c75d0bf16300b9721e",
+
+    yagnaOptions: { apiKey: "try_golem" },
+    budget: 0.5,
+    networkIp: "192.168.0.0/24",
+    // Control the execution of tasks
+    //maxTaskRetries: 2,
+    maxParallelTasks: 2,
+    taskTimeout: 5 * 60 * 1000, // 5 min
+    // Useful for debugging
+    logLevel: LogLevel.Info,
+  });
+
+  let serverIP;
+  let runClient = true;
+
+  try {
+    // Your code goes here
+
+    let result = executor.run(async (ctx) => {
+      console.log("Netcat provider deployed");
+      serverIP = ctx.getIp();
```

Comment:
`.getNetworkId, .getIp()`
`ctx.getNetwork().getIp(), ctx.getNetwork().getId()`

Reply:
created an issue for it: https://golemproject.atlassian.net/jira/software/c/projects/JST/issues/JST-694